MediaAdminBundle/Resources/public/less/groupFolderTreePanel.less

```diff
@@ -0,0 +1,55 @@
+#tab-media_folder_tree {
```

Comment:
this css is almost the same that for the nodeTree, may be can you create a generic css for the two entities